redis_commands.c

```diff
@@ -1616,6 +1619,10 @@ int redis_set_cmd(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
         redis_cmd_append_sstr(&cmdstr, set_type, strlen(set_type));
     if (keep_ttl)
         redis_cmd_append_sstr(&cmdstr, "KEEPTTL", 7);
+    if (get) {
+        REDIS_CMD_APPEND_SSTR_STATIC(&cmdstr, "GET");
+        *ctx = redis_sock;
```

Comment:
Maybe instead of setting this to the `redis_sock` address, we could use a poison value, so it's immediately obvious what's happening if we accidentally free it.

e.g. `0xdeadbeef` or `0xfafafafafa`

Reply:
OK, I will change the value after merge